sdk/dotnet/State/ArtifactoryRemoteStateReference.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2016-2020, Pulumi Corporation.
```

Comment:
how was this file generated?

Reply:
This whole PR is manually written

Reply:
There is currently no way to generate these since the Terraform provider is not bridged. The same approach was used for TypeScript and Python. Fortunately they do not change very often, however!